go.mod

```diff
@@ -4,119 +4,142 @@ module github.com/cosmos/ibc-go/v3
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
+replace github.com/zondax/hid => github.com/zondax/hid v0.9.0
```

Comment:
I had to add this [to make the branch build on my Mac running Big Sur](https://github.com/cosmos/cosmos-sdk/issues/11364). I will take it off before merging.